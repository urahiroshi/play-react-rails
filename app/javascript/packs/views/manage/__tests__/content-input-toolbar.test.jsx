import React from 'react'
import renderer from 'react-test-renderer'
import ContentInputToolbar from '../content-input-toolbar'

describe('#render', () => {
  it('should show snapshot', () => {
    const component = renderer.create(<ContentInputToolbar />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
