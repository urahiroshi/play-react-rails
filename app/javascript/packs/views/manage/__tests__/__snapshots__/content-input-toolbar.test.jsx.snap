// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<div
  className="btn-toolbar manage-content-toolbar"
>
  <div
    className="btn-group"
  >
    <button
      className="btn btn-default manage-bold-button"
      onClick={[Function]}
      type="button"
    >
      B
    </button>
    <button
      className="btn btn-default manage-italic-button"
      onClick={[Function]}
      type="button"
    >
      I
    </button>
  </div>
  <div
    className="btn-group"
  >
    <button
      className="btn btn-default"
      onClick={[Function]}
      type="button"
    >
      H1
    </button>
    <button
      className="btn btn-default"
      onClick={[Function]}
      type="button"
    >
      H2
    </button>
    <button
      className="btn btn-default"
      onClick={[Function]}
      type="button"
    >
      H3
    </button>
    <button
      className="btn btn-default"
      onClick={[Function]}
      type="button"
    >
      H4
    </button>
    <button
      className="btn btn-default"
      onClick={[Function]}
      type="button"
    >
      H5
    </button>
  </div>
  <div
    className="btn-group"
  >
    <button
      className="btn btn-default"
      onClick={[Function]}
      style={
        Object {
          "color": "#FF0000",
        }
      }
      type="button"
    >
      A
    </button>
  </div>
</div>
`;
